core/string/ustring.cpp

```diff
@@ -5286,7 +5286,7 @@ bool String::is_valid_html_color() const {
 }
 
 // Changes made to the set of invalid filename characters must also be reflected in the String documentation for is_valid_filename.
-static const char *invalid_filename_characters = ": / \\ ? * \" | % < >";
+static const char *invalid_filename_characters[] = { ":", "/", "\\", "?", "*", "\"", "|", "%", "<", ">" };
```

Comment:
`static char invalid_filename_characters[] = { ':', '/', '\\', '?', '*', '"', '|', '%', '<', '>' };`

8x smaller( for x64 ), now fits in one cache line

Reply:
no equivalent function taking a size ? Too bad. Always better than forced null character ending

Reply:
Well, to keep in mind then. If you move forward on your no null terminated strings proposal, they will have to be added anyway.

Reply:
Totally agree. I've been working towards size based string inputs with #99806 and https://github.com/godotengine/godot-proposals/issues/11249, feel free to voice your support :)